### PR TITLE
LUN-138: Resusable-SingleChoiceListItem

### PR DESCRIPTION
**Pull request by f1sh1918**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

I decided to keep this refactoring quite small because we have to keep the color logic in the component because its depending on state. I could have made a `SingleChoiceItemWrapper` but it's not worth in my opinion. To use the `ListItem` would mean to but a lot of additional logic inside which would make the component itself too complicated. Feel free for discussion if there are better proposals for refactoring

### CODE REVIEW

**Review by SteffiStuffel — APPROVED** [review]
Looks good. Tested on android

**Comment by SteffiStuffel** [comment]
And please delete other not used LUN-138 branch if no longer needed.

**Review by sarahsporck — APPROVED** [review]
Tested on Android emulator

File: src/components/SingleChoiceListItem.tsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
 import React, { useState } from 'react'
```
[the remark this answers is not in the document]
Comment (f1sh1918) [reply src/components/SingleChoiceListItem.tsx:1]: @SteffiStuffel  VocabularyListItem not sure if thats way better :)

[the remark this answers is not in the document]
Comment (f1sh1918) [reply src/components/SingleChoiceListItem.tsx]: renamed to `WordItem`

**Pull request merged**